From: Petr Tesarik <ptesarik@suse.com>
Subject: Add a fallback PN_XNUM definition
Upstream: never, compatibility with old products

Distributions before SLE11 did not define PN_XNUM in <elf.h>.

Signed-off-by: Petr Tesarik <ptesarik@suse.com>
---
 netdump.h |    4 ++++
 1 file changed, 4 insertions(+)

--- a/netdump.h
+++ b/netdump.h
@@ -19,6 +19,10 @@
 #include <elf.h>
 #include "vmcore.h"
 
+#ifndef PN_XNUM
+#define PN_XNUM		0xffff
+#endif
+
 #define MIN_NETDUMP_ELF32_HEADER_SIZE \
         sizeof(Elf32_Ehdr)+sizeof(Elf32_Phdr)+sizeof(Elf32_Phdr)
 #define MIN_NETDUMP_ELF64_HEADER_SIZE \
